src/utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Pads the front of the given hex string with zeroes until it reaches the
+ * target length. If the input string is already longer than or equal to the
+ * target length, it is returned unmodified.
+ *
+ * If the input string is "0x"-prefixed or not a hex string, an error will be
+ * thrown.
+ *
+ * @param hexString The hexadecimal string to pad with zeroes.
+ * @param targetLength The target length of the hexadecimal string.
```

Comment:
Nit: We typically separate parameter descriptions from the variable name with a ` - `, as is done in the JSDoc docs (though it seems to be optional according to JSDoc IIRC)

```suggestion
 * @param hexString - The hexadecimal string to pad with zeroes.
 * @param targetLength - The target length of the hexadecimal string.
```

Reply:
Yeah, I think it is optional, but you are correct that it is the convention!